Jenkinsfile

```diff
@@ -1,8 +1,8 @@
-@Library('xmos_jenkins_shared_library@master') _
+@Library('xmos_jenkins_shared_library@develop') _
 getApproval()
 pipeline {
   agent {
-    label 'x86&&macOS&&Apps'
+    label 'x86 && macOS && brew'    
```

Comment:
Does this Pipeline have to run on macOS? Could the label be `x86_64&&brew` to enable the use of more agents?